packages/indexer/src/simple-indexer.ts

```diff
@@ -0,0 +1,19 @@
+import { BaseIndexer } from "./indexer";
+
+export class SimpleIndexer extends BaseIndexer {
+  private taskHandle?: Timer;
+
+  async start(): Promise<void> {
+    const startingHeight = await this.getCurrentHeight();
+
+    this.taskHandle = setInterval(() => this.indexForever(startingHeight), 5000);
```

Comment:
Add interval to config

Maybe `SimpleIndexer` should have its own config opts?